### PR TITLE
fix: remove main tag in `User.js` causing duplicate

### LINKED ISSUE #651: Page can only have one main tag
### Description

On the profile page there is a main tag nested inside of another main tag when there should only be one.

### Screenshots

Screenshot 2021-11-09 143451

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Sfitzg**
Removed the main element in `User.js` as this was causing a duplicate main element

Closes #651 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/661"></a>

### CODE REVIEW

**Review by mohan-murali — APPROVED** [review]

**Comment by mohan-murali** [comment]
maybe you can try fix the failing test?

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Can you provide an ss of the inspector?

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Looking at the structure of the code, I would suggest main be present in the home component and user component as these are the two main routes. Having main in too many child components complicates the structure and will lead to further problems later of main being duplicated again.

**Comment by Sfitzg** [comment]
@emmalearnscode I agree with you but the structure was pretty confusing and I also feel that the main nav should be outside of the main area as it is consistent across pages?

**Review by krishguptadev — COMMENTED** [review]
## Merge Conflict

Thank you for your contribution. Your pull request currently has conflicting changes with our `main` branch. In order to review this pull request, we will need those conflicts to be resolved.

We recommend doing this with the `git rebase` command. If you are unsure of how to fix this, let us know and we would be happy to assist you.

**Comment by emmalearnscode** [comment]
> @emmalearnscode I agree with you but the structure was pretty confusing and I also feel that the site nav should be outside of the main area as it is consistent across pages?
> 
> I'm happy to amend but I also feel we should look at pulling the main navigation out of the main area? maybe in another issue ? Happy to hear your thoughts
> 
> thanks

Oh interesting, I hadn't noticed that the navbar was inside different components like that. Yes, ideally the navbar is outside of the routes so we should probably raise another issue.

File: src/Components/UserProfile/User.js [hunk 1]
```diff
@@ -26,21 +26,23 @@ function User() {
...
         }
       />
-      {showProgress && <ProgressBar mode="indeterminate" />}
-      {skeleton && <Placeholder />}
-      {error && <ErrorPage />}
-      {!error && !skeleton && (
-        <ProfilePage profile={profile} username={username} />
-      )}
-    </main>
+      <main>
```
Comment (krishguptadev) [inline src/Components/UserProfile/User.js:37]: Still a main tag here 😅

**Review by krishguptadev — CHANGES_REQUESTED** [review]

File: src/Components/UserProfile/ErrorPage.js [hunk 2]
```diff
@@ -4,10 +4,24 @@ function ErrorPage() {
...
+        <img
+          src="/eddiehub_community_logo.webp"
+          alt="image"
+          style={{ width: '150px' }}
+        />
         <h1>Profile not found.</h1>
         <h1>If you are a new user, please consider registering at LinkFree.</h1>
-        <h2>Read the documentation <a href="https://github.com/EddieHubCommunity/LinkFree#readme" target="_blank" rel="noreferrer">here</a>.</h2>
+        <h2>
+          Read the documendation{' '}
```
Comment (emmalearnscode) [inline src/Components/UserProfile/ErrorPage.js:15]: This reintroduces a spelling mistake in documentation.

File: src/Components/UserProfile/User.js (hunk 1 above)
Comment (emmalearnscode) [reply src/Components/UserProfile/User.js:37]: I understand why there is a main tag here, you are moving the navbar outside the main content which is correct. Can you take a screenshot of the outputted html code to make so we can see that it solves the issue and another main tag doesn’t sneak in from any other component.

**Comment by Sfitzg** [comment]
Attached are the Screenshots 
Hope this is all in order now 😄 


## Home Page
Homepage

## Profile Page

UserProfile

## Error Page
Errorpage

**Review by krishguptadev — APPROVED** [review]

**Review by qwe123coder — APPROVED** [review]
LGTM! :rocket:

**Review by Vyvy-vi — APPROVED** [review]
Looks good to me!

Thank you for opening this PR, and contributing to LinkFree! :tada:

**Pull request merged**